tests/model/test_Model.py

```diff
@@ -0,0 +1,76 @@
+import unittest
```

Comment:
Could you order the imports according to https://realpython.com/absolute-vs-relative-python-imports/

basically group into 
- Standard library imports
- Third party imports
- Local application imports

It's the conventional way apparently